.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
```

Comment:
Do you know if setting this will affect development on Windows?  If their git config has `core.autocrlf` set to True?

I'm hesitant to enforce an `.editorconfig` without checking on some of these nuances (and also probably enforcing with `.gitattributes`).

Maybe this should be a separate PR

Reply:
> Upon commit, git will convert any CRLF back to LF, but will not touch LF endings, afaict.


Thanks for the explanation.  I like the idea of it staying set as `end_of_line = lf` in the editorconfig.  

Devs working in Windows environments are forced to deal with the `crlf` issue on their own.  I just didn't want that to interfere with anything or introduce an extra burden in those cases.